addon/PlacesProvider.js

```diff
@@ -782,7 +782,6 @@ Links.prototype = {
                     FROM moz_bookmarks b, moz_places p
                     WHERE type = :type
                     AND b.fk = p.id
-                    AND p.last_visit_date IS NOT NULL
```

Comment:
Removing this condition will result in `place:folder=TOOLBAR` type bookmarks to show up in the query, so those will need to be additionally handled if things aren't expecting `place:` `URI`s.